py3status/modules/file_status.py

```diff
@@ -68,16 +92,40 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
+        # init data
+        file_status_data = {}
+        # expand glob from paths
+        paths = list(map(glob, self.path))
+        # merge list of paths
+        paths = [x for x in chain.from_iterable(paths)]
+
+        # fill data
+        file_status_data['icon'] = self.icon_unavailable
+        color = self.py3.COLOR_BAD
+
+        if paths:
+            file_status_data['icon'] = self.icon_available
```

Comment:
Here, I really would like to deprecate `icon` in favor of a `files` (like `message`/`messages`) so the format could look like this... `format = '\?if=files ●|■'` instead... And `{files}` would be a number of files.

I guess we stick with just `allow several path` and not rewriting this from scratch.  lol.

Just mentioning that `file`_status will pick up directories too with wildcard. Anyhow, keep it as-is.

Reply:
So i remove legacy stuff?

Reply:
Ideally, we do no more than to `allow several path` PR. We definitely can add (count) `{paths}` for  `format` because it is still related.

It'd be good to ask the fam first about deprecating `{icon}` as this would allow us to get rid of `icon_available` and `icon_unavailable` in favor of `format = '\?if=paths ●|■'`. Also, ask about the new `{format_path}` in default `format` (versus) just icons. I think it's better than icons too.